Source/Scene/GlobeSurfaceTileProvider.js

```diff
@@ -160,6 +162,13 @@ define([
         this._baseColor = undefined;
         this._firstPassInitialColor = undefined;
         this.baseColor = new Color(0.0, 0.0, 0.5, 1.0);
+
+        /**
+         * Gets or sets the array of clipping planes, defined in world space;
```

Comment:
Tweak this description too.

Reply:
Maybe this should be marked `@private`. Otherwise it should be mentioned in `CHANGES.md`.

Reply:
I marked this private, I don't think this should ever need to be set at the tileProvider level.